src/lib/timing.c

```diff
@@ -149,13 +174,15 @@ void xdebug_nanotime_init(void)
 #endif
 
 	context.start_abs = xdebug_get_nanotime_abs(&context);
-	context.last_abs = 0;
 #if PHP_WIN32 | __APPLE__ | defined(CLOCK_MONOTONIC)
 	context.start_rel = xdebug_get_nanotime_rel(&context);
-	context.last_rel = 0;
 #endif
 
 	XG_BASE(nanotime_context) = context;
+
+#if WIN_SUPPORTS_RDTSC
+	xdebug_nanotime_init_calibrate_rdtsc();
```

Comment:
Calibration is needed, but we can make it optional based on some env. variable/configuration flag. Please advise.

Reply:
Why CI outputs:
```
 ========DIFF========
001+ Xdebug: [Config] Invalid mode 'develop,profiler' set for 'xdebug.mode' configuration setting (See: https://xdebug.org/docs/errors#CFG-C-MODE)
========DONE========
FAIL xdebug_time_index_with_profiler.phpt() [tests/develop/xdebug_time_index_with_tsc_clock.phpt] 
```

when multiple modes - documented in https://xdebug.org/docs/all_settings - should be supported?